wolox-books/src/app/screens/auth/auth.guard.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { Observable } from 'rxjs';
+import { UserService } from '../../components/user/user.service';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthGuard implements CanActivate {
+
+  constructor(private userService: UserService, private router: Router) {  }
+
+  public canActivate() {
+    if(this.userService.loggedIn()){
+      return true;
+    } else {
```

Comment:
No need for the `else`, if not true, either way it's going to execute the lines in here 😄